README.md

```diff
@@ -43,56 +15,37 @@ operations:
 The convolution operations have several implementations, including tiled and
 Winograd kernels. The supported data format is NHWC.
 
-## Citation
+The project is maintained by [Codeplay Software](https://developer.codeplay.com).
 
-If you use SYCL-DNN in your research, please cite the library as follows:
+## Supported Platforms
 
-> Rod Burns, John Lawson, Duncan McBain, and Daniel Soutar. 2019. *Accelerated
-> Neural Networks on OpenCL Devices Using SYCL-DNN.* In Proceedings of the
-> International Workshop on OpenCL (IWOCL'19). ACM, New York, NY, USA, Article
-> 10, 4 pages. DOI: https://doi.org/10.1145/3318170.3318183
-
-```bibtex
-@inproceedings{Burns:2019:ANN:3318170.3318183,
- author = {Burns, Rod and Lawson, John and McBain, Duncan and Soutar, Daniel},
- title = {Accelerated Neural Networks on OpenCL Devices Using SYCL-DNN},
- booktitle = {Proceedings of the International Workshop on OpenCL},
- series = {IWOCL'19},
- year = {2019},
- isbn = {978-1-4503-6230-6},
- location = {Boston, MA, USA},
- pages = {10:1--10:4},
- articleno = {10},
- numpages = {4},
- url = {http://doi.acm.org/10.1145/3318170.3318183},
- doi = {10.1145/3318170.3318183},
- acmid = {3318183},
- publisher = {ACM},
- address = {New York, NY, USA},
- keywords = {GPGPU, OpenCL, SYCL, machine learning, neural networks},
-}
-```
-
-## Requirements
+The master branch of SYCL-DNN is regularly tested with the "Supported" hardware 
+listed on [the ComputeCpp Supported Platforms page](https://developer.codeplay.com/products/computecpp/ce/guides/platform-support).
 
+SYCL-DNN may also work on other hardware and platforms assuming they implement 
+ SPIR or SPIR-V support.
+ 
 SYCL-DNN is primarily tested on Ubuntu 16.04 LTS with the corresponding
-default package versions, as summarized below.
+default package versions.
 
-* SYCL-DNN will generally match the most recently released ComputeCpp, though
+SYCL-DNN will generally match the most recently released ComputeCpp, though
   it is likely to be compatible with other versions. We test against the most
   recent version.
 
-* Hardware capable of running SYCL code. For ComputeCpp this could be OpenCL
-  1.2-capable hardware and drivers with SPIR 1.2/SPIR-V/PTX support.
+## Getting Started with SYCL-DNN
 
-* A C++-11 compiler and STL implementation. We test against GCC 5.4.0.
-
-* CMake. Tested against version 3.5.1.
+### Pre-requisites 
 
+* CMake (version 3.5.1 and above)
```

Comment:
Nit: Version 3.5.1 is not the minimum version, just the version we test with. It will likely work with 3.4, possibly even earlier versions.

Reply:
My logic is that I went with the minimum requirements we list for ComputeCpp. If people want to try older versions then they can but we might as well be up front about what we are prepared to support.